skt/executable.py

```diff
@@ -59,12 +59,19 @@ def report_results(result_path, result_string, report_path, report_string):
         report_path:    A full path of report log.
         report_string:  The content of string report.
     """
-    with os.fdopen(os.open(result_path, os.O_CREAT | os.O_WRONLY),
-                   'w') as result_file:
-        result_file.write(result_string)
-    with os.fdopen(os.open(report_path, os.O_CREAT | os.O_WRONLY),
-                   'w') as report_file:
-        report_file.write(report_string)
+    try:
```

Comment:
What's the idea behind the try/except block? If the result files can't be created, it a problem in user's configuration and we should hide it.

Reply:
@veruu I am little bit confused. Could you ask you about idea behind the try/except block here? https://github.com/RH-FMK/skt/blob/master/skt/kerneltree.py#L54. Thanks

Reply:
Sorry I make you confused. Could you elaborate `user's configuration` ? I point out the link of the code because I think try/except for 2 situations are the same. I think the result file cannot be created being the same the the workdir cannot create.

Reply:
The workdir can't be created if the user created it manually before running skt. We are always removing any result files so not being able to create a result file because it already exists would never happen.

Reply:
Ah I see. the user has to resolve and it cannot pass in exception. But the workdir is different because it could probably pass if user created it manually before. 
What about the next commit?

Reply:
Functionality-wise, that one is OK. I'd like to see the "source_dir" argument renamed to "output_dir" or something similar (to not be confused with kernel source directory), and rename the function itself ("exresult" is not that clear - spelling out the full word or using "old" would work better, for example)